device.c

```diff
@@ -297,11 +297,11 @@ static int vcam_enum_frameintervals(struct file *file,
     fival->type = V4L2_FRMIVAL_TYPE_STEPWISE;
     frm_step = &fival->stepwise;
     frm_step->min.numerator = 1001;
-    frm_step->min.denominator = 60000;
+    frm_step->min.denominator = 60060;
```

Comment:
Can you explain?

Reply:
Actually, it just changes `frm_step->max.denominator` to 1001 will be okay.

Reply:
Always minimize the necessary changes.